AdyenActions/Actions/RedirectAction.swift

```diff
@@ -18,15 +18,23 @@ public struct RedirectAction: Decodable {
     /// Native redirect data.
     public let nativeRedirectData: String?
     
+    internal let paymentMethodType: String?
+    
     /// Initializes a redirect action.
     ///
     /// - Parameters:
     ///   - url: The URL to which to redirect the user.
     ///   - paymentData: The server-generated payment data that should be submitted to the `/payments/details` endpoint.
     ///   - nativeRedirectData: Native redirect data.
```

Comment:
The comment is missing the `paymentMethodType` info

```suggestion
    ///   - nativeRedirectData: Native redirect data.
    ///   - paymentMethodType: The type of the payment method.
```